rpc/params/charms.go

```diff
@@ -116,10 +116,8 @@ type CharmMeta struct {
 	ExtraBindings  map[string]string            `json:"extra-bindings,omitempty"`
 	Categories     []string                     `json:"categories,omitempty"`
 	Tags           []string                     `json:"tags,omitempty"`
-	Series         []string                     `json:"series,omitempty"`
 	Storage        map[string]CharmStorage      `json:"storage,omitempty"`
 	Devices        map[string]CharmDevice       `json:"devices,omitempty"`
-	Deployment     *CharmDeployment             `json:"deployment,omitempty"`
```

Comment:
Officially this is a breaking change, do we know if it's a real breaking change?

Reply:
This breaking change would fall under the umbrella of the breaking change removing support for pod spec charms, right?